webservice/src/main/java/com/hortonworks/registries/webservice/RegistryApplication.java

```diff
@@ -94,23 +94,24 @@ public void run(RegistryConfiguration registryConfiguration, Environment environ
     }
 
     private void initializeUGI(RegistryConfiguration conf) throws IOException {
-        LOG.debug("Initialization of User Group ininformation...");
+        LOG.debug("Initialization of User Group information...");
         if (UserGroupInformation.isSecurityEnabled()) {
```

Comment:
This check i believe requires core-site.xml. Can we not use the conf.getServiceAuthenticationConfiguration() itself?